### PR TITLE
Changed split function, which reads new search log

### PR DESCRIPTION
**Pull request by ogolovatyi**
Format we have  10.20.30.20!10.20.30.20 was not parsed with error "web_log.log is not a search log or partnerlog".
It is fixed now.

### CODE REVIEW

**Comment by macpd** [comment]
In what version did the the log format change? Will this break parsing of old search logs? If so, please make this configurable with a flag (defaulting to new behavior)

**Comment by ogolovatyi** [comment]
Both 7.2 and 7.4 search logs looks like :
10.17.10.10!10.17.10.10 - - [21/Oct/2015:08:45:57 -0800] changes done to distinguish both 10.17.10.10 , as it was initially, and 10.17.10.10!10.17.10.10 as search log.

File: searchstats.py [hunk 1]
```diff
@@ -115,7 +115,8 @@ def main():
   for line in search_log:    # Iterate through lines of the log file.
     if first_line:    # If this is the first line, we need to figure out what
                       # kind of file we've received (search log or partnerlog).
-      if len(((line.split())[0]).split('.')) == 4:
+      if (len(((line.split())[0]).split('.')) == 7)or(len(((line.split())[0]).split('.')) == 4):
```
Comment (macpd) [inline searchstats.py]: please add a spaces on either side of the "or" and split into 2 lines (for readability)

File: searchstats.py [hunk 2]
```diff
@@ -115,7 +115,8 @@ def main():
...
+        # True for both formats 10.10.10.10!10.10.10.10 -- [21/Oct/2015:08:45:57 -0800] and 10.10.10.10 -- [21/Oct/2015:08:45:57 -0800]
```
Comment (macpd) [inline searchstats.py]: please add quotes around the values, and format to not exceed 80 chars per line

File: searchstats.py [hunk 3]
```diff
@@ -115,7 +115,12 @@ def main():
   for line in search_log:    # Iterate through lines of the log file.
     if first_line:    # If this is the first line, we need to figure out what
                       # kind of file we've received (search log or partnerlog).
-      if len(((line.split())[0]).split('.')) == 4:
+      if (len(((line.split())[0]).split('.')) == 7)or(
```
Comment (macpd) [inline searchstats.py]: please add a space on both sides of "or"

**Comment by michaelcizmar** [comment]
Any reason why this can't be merged?

**Comment by michaelcizmar** [comment]
Thanks!

**Pull request merged**